test/ndp-router-advertisement/src/nf.rs

```diff
@@ -0,0 +1,111 @@
+use colored::*;
+use netbricks::headers::*;
+use netbricks::operators::*;
+use std::str::FromStr;
+
+struct Meta {
+    payload_len: u16,
+}
+
+pub fn ndp_nf<T: 'static + Batch<Header = NullHeader>>(parent: T) -> CompositionBatch {
+    let pipeline = parent
+        .parse::<MacHeader>()
+        .filter(box |pkt| match pkt.get_header().etype() {
+            Some(EtherType::IPv6) => true,
+            _ => false,
+        });
+
+    ndp_router_advertisementertisement_nf(pipeline)
+}
+
+#[inline]
+fn ndp_router_advertisementertisement_nf<T: 'static + Batch<Header = MacHeader>>(
```

Comment:
advertisementertisementertisementertisementment

Reply:
renamed it to advertisementertisementertisementertisementmentadvertisementertisementertisementertisementmentadvertisementertisementertisementertisementment. :)